{
  "id": "CVE-2024-12619",
  "sourceIdentifier": "cve@gitlab.com",
  "published": "2025-03-28T10:15:15.760",
  "lastModified": "2025-03-28T10:15:15.760",
  "vulnStatus": "Received",
  "descriptions": [
    {
      "lang": "en",
      "value": "An issue has been discovered in GitLab CE/EE affecting all versions from 16.0 before 17.8.6, 17.9 before 17.9.3, and 17.10 before 17.10.1, allowing internal users to gain unauthorized access to internal projects."
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "cve@gitlab.com",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:H/UI:R/S:U/C:H/I:L/A:N",
          "attackVector": "NETWORK",
          "attackComplexity": "LOW",
          "privilegesRequired": "HIGH",
          "userInteraction": "REQUIRED",
          "scope": "UNCHANGED",
          "confidentialityImpact": "HIGH",
          "integrityImpact": "LOW",
          "availabilityImpact": "NONE",
          "baseScore": 5.2,
          "baseSeverity": "MEDIUM"
        },
        "exploitabilityScore": 0.9,
        "impactScore": 4.2
      }
    ]
  },
  "weaknesses": [
    {
      "source": "cve@gitlab.com",
      "type": "Primary",
      "description": [
        {
          "lang": "en",
          "value": "CWE-1220"
        }
      ]
    }
  ],
  "references": [
    {
      "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/509324",
      "source": "cve@gitlab.com"
    },
    {
      "url": "https://hackerone.com/reports/2888260",
      "source": "cve@gitlab.com"
    }
  ]
}